iterx_test.go

```diff
@@ -480,12 +480,8 @@ func TestIterxUnsafe(t *testing.T) {
 	})
 
 	t.Run("select default unsafe", func(t *testing.T) {
-		gocqlx.DefaultUnsafe = true
-		defer func() {
-			gocqlx.DefaultUnsafe = false
-		}()
 		var v []UnsafeTable
-		err := session.Query(stmt, nil).Iter().Select(&v)
+		err := session.Query(stmt, nil).Unsafe().Iter().Select(&v)
 		if err != nil {
 			t.Fatal("Select() failed:", err)
 		}
```

Comment:
What was the purpose of the removed code with `defer` and why can we remove it now?

Reply:
Ah, is it because we migrate `DefaultUnsafe` from being a global gocqlx attribute to be per-query attribute?

Reply:
> Ah, is it because we migrate `DefaultUnsafe` from being a global gocqlx attribute to be per-query attribute?

Exactly